Index: .idea/compiler.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/compiler.xml b/.idea/compiler.xml
--- a/.idea/compiler.xml	
+++ b/.idea/compiler.xml	
@@ -3,9 +3,6 @@
   <component name="CompilerConfiguration">
     <annotationProcessing>
       <profile default="true" name="Default" enabled="true" />
-<<<<<<< HEAD
-    </annotationProcessing>
-=======
       <profile name="Maven default annotation processors profile" enabled="true">
         <sourceOutputDir name="target/generated-sources/annotations" />
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
@@ -21,6 +18,5 @@
     <option name="ADDITIONAL_OPTIONS_OVERRIDE">
       <module name="realestatesite" options="-parameters" />
     </option>
->>>>>>> origin/main
   </component>
 </project>
\ No newline at end of file
Index: server/target/generated-sources/annotations/com/example/realestatesite/mappers/UserMapperImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/target/generated-sources/annotations/com/example/realestatesite/mappers/UserMapperImpl.java b/server/target/generated-sources/annotations/com/example/realestatesite/mappers/UserMapperImpl.java
--- a/server/target/generated-sources/annotations/com/example/realestatesite/mappers/UserMapperImpl.java	
+++ b/server/target/generated-sources/annotations/com/example/realestatesite/mappers/UserMapperImpl.java	
@@ -8,10 +8,8 @@
 
 @Generated(
     value = "org.mapstruct.ap.MappingProcessor",
-    date = "2023-05-09T19:41:41+0600",
+    date = "2023-05-12T16:11:57+0600",
     comments = "version: 1.5.3.Final, compiler: javac, environment: Java 18.0.2 (Oracle Corporation)"
-    date = "2023-05-12T02:06:45+0600",
-    comments = "version: 1.5.3.Final, compiler: javac, environment: Java 17.0.6 (Private Build)"
 )
 public class UserMapperImpl implements UserMapper {
 
Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	
+++ b/.idea/misc.xml	
@@ -1,6 +1,13 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" default="true">
+  <component name="MavenProjectsManager">
+    <option name="originalFiles">
+      <list>
+        <option value="$PROJECT_DIR$/server/pom.xml" />
+      </list>
+    </option>
+  </component>
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_18" default="true" project-jdk-name="openjdk-18" project-jdk-type="JavaSDK">
     <output url="file://$PROJECT_DIR$/out" />
   </component>
   <component name="ProjectType">
Index: server/target/generated-sources/annotations/com/example/realestatesite/mappers/AnnouncementMapperImpl.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/server/target/generated-sources/annotations/com/example/realestatesite/mappers/AnnouncementMapperImpl.java b/server/target/generated-sources/annotations/com/example/realestatesite/mappers/AnnouncementMapperImpl.java
--- a/server/target/generated-sources/annotations/com/example/realestatesite/mappers/AnnouncementMapperImpl.java	
+++ b/server/target/generated-sources/annotations/com/example/realestatesite/mappers/AnnouncementMapperImpl.java	
@@ -8,10 +8,8 @@
 
 @Generated(
     value = "org.mapstruct.ap.MappingProcessor",
-    date = "2023-05-09T19:41:41+0600",
+    date = "2023-05-12T16:11:58+0600",
     comments = "version: 1.5.3.Final, compiler: javac, environment: Java 18.0.2 (Oracle Corporation)"
-    date = "2023-05-12T02:06:45+0600",
-    comments = "version: 1.5.3.Final, compiler: javac, environment: Java 17.0.6 (Private Build)"
 )
 public class AnnouncementMapperImpl implements AnnouncementMapper {
 
